docs/tutorial.md

```diff
@@ -0,0 +1,721 @@
+                           </imageView>
+                           <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Hello Blockstack iOS" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="Wfj-A6-BZM">
+                               <rect key="frame" x="108" y="432" width="158" height="21"/>
+                               <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                               <nil key="textColor"/>
+                               <nil key="highlightedColor"/>
+                           </label>
+                       </subviews>
+                       <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                       <constraints>
+                           <constraint firstItem="Wfj-A6-BZM" firstAttribute="centerX" secondItem="6Tk-OE-BBY" secondAttribute="centerX" id="AZy-qf-xHq"/>
+                           <constraint firstItem="Wfj-A6-BZM" firstAttribute="top" secondItem="6Tk-OE-BBY" secondAttribute="top" constant="412" id="SwP-qV-1RP"/>
+                           <constraint firstItem="SpU-hA-y2f" firstAttribute="centerX" secondItem="6Tk-OE-BBY" secondAttribute="centerX" id="XdI-Db-fDo"/>
+                           <constraint firstItem="SpU-hA-y2f" firstAttribute="top" secondItem="6Tk-OE-BBY" secondAttribute="top" constant="253" id="xc5-po-W1E"/>
+                       </constraints>
+                       <viewLayoutGuide key="safeArea" id="6Tk-OE-BBY"/>
+                   </view>
+               </viewController>
+               <placeholder placeholderIdentifier="IBFirstResponder" id="iYj-Kq-Ea1" userLabel="First Responder" sceneMemberID="firstResponder"/>
+           </objects>
+           <point key="canvasLocation" x="52" y="374.66266866566718"/>
+       </scene>
+   </scenes>
+   ```
+
+9. Immediately after scenes but before the close of the `</document>` tag add the following `<resources>`.
+
+   ```xml
+		   <resources>
+         <image name="Image" width="64" height="64"/>
+		   </resources>
+		</document>
+   ```
+
+10. Choose **Run > Run app** in the emulator.
+
+	  The emulator now contains a new splash screen.
+
+	  ![](images/splash.png)
+
+
+### Add a Main.storyboard
+
+Rather than have you build up your own UI, this section has you copy and paste a layout into the XML file source code for the **Main.storyboard** file.
+
+1.  Select the `Main.storyboard` file.
+2.  Chooose **Open As > Source Code**
+
+    The `blockstack-example/blockstack-example/Base.lproj/Main.storyboard` file
+    defines the graphical elements. Some elements are required before you can
+    functionality to your  code.
+
+3. Replace the <view> element with the following:
```

Comment:
The "view" doesn't actually show on the tutorial, so it reads as: "Replace the element with the following:" which led me to replace all the XML and break main.storyboard. Probably caused by the "<" character, so you can just replace it with: "replace the `view` element under `viewController` with the following:"

Reply:
✔️